fullstop-timemachine/src/test/java/org/zalando/stups/fullstop/swagger/api/FullstopApiTest.java

```diff
@@ -129,6 +130,19 @@ public void testViolations() throws Exception {
     }
 
     @Test
+    public void testViolationsWithParams() throws Exception {
+
+        when(violationServiceMock.queryViolations(any(List.class), any(DateTime.class), eq(0L), eq(true), any()))
+                .thenReturn(new PageImpl<>(newArrayList(violationResult)));
+
+        ResultActions resultActions = this.mockMvc.perform(
+                get("/api/violations?accounts=123&checked=true&last-violation=0&since=" + new DateTime(UTC)))
```

Comment:
does `new DateTime(UTC).toString()` really produce the desired format "yyyy-MM-dd...."? You should use a real here. Luckily DateTime provides a method `toString(String)` where you can pass your format as parameter.